tests/testthat/test-clean-names.R

```diff
@@ -93,4 +93,59 @@ test_that("Tests for cases beyond default snake", {
 
 test_that("errors if not called on a data.frame", {
   expect_error(clean_names(1:3), "clean_names() must be called on a data.frame.  Consider janitor::make_clean_names() for other cases of manipulating vectors of names.", fixed = TRUE)
-})
\ No newline at end of file
+})
+
+
+#------------------------------------------------------------------------------# 
+#---------------------------- Tests for sf method -----------------------------#
```

Comment:
@sfirke here are the tests for the sf method of `clean_names()`. I just changed the`test_df` to be a sf object and ran the same tests. Everything checks out :) 👍